scripts/systemd.sh

```diff
@@ -41,12 +41,6 @@ function doInstallConfig
   fi
   sudo mkdir -p /etc/athens
   sudo install -v -o root -g root -m 644 config.toml /etc/athens
-
-  # if storage is on disk, this is where the database goes (see scripts/service/athens.service)
-  ATHENS_DISK_STORAGE_ROOT=/var/run/athens
-  sudo mkdir -p $ATHENS_DISK_STORAGE_ROOT
-  sudo chown www-data $ATHENS_DISK_STORAGE_ROOT
-  sudo chgrp www-data $ATHENS_DISK_STORAGE_ROOT
```

Comment:
Shouldn't the `chown` and `chgrp` commands be kept in order to still make sure `www-data` owns the `ReadWritePaths`?